ha/setup/ha_setup.py

```diff
@@ -967,6 +976,100 @@ def remove_config_files(self):
         for file in files:
             CleanupCmd.remove_file(file)
 
+    def remove_consul_keys(self, node_name:str, first_in_lexicographical:bool = False) -> None:
+        """
+        Remove consul keys for the node
+        Args:
+            node_name:
+            first_in_lexicographical:
+
+        Returns:
+            None
+        """
+        # remove consul keys for current node
+        self._confstore.delete(f"{const.CLUSTER_CONFSTORE_NODES_KEY}/{node_name}")
+        # delete node_map key
+        self._confstore.delete(f"{const.PVTFQDN_TO_NODEID_KEY}/{node_name}")
+        # remove it from system health
+        machine_id = self.get_machine_id()
+        system_health_obj = SystemHealth(self._confstore)
```

Comment:
node_id = Conf.get(self._index, f"server_node{_DELIM}{machine_id}{_DELIM}node_id")
        cluster_id = Conf.get(self._index, f"server_node{_DELIM}{machine_id}{_DELIM}cluster_id")
        site_id = Conf.get(self._index, f"server_node{_DELIM}{machine_id}{_DELIM}site_id")
        rack_id = Conf.get(self._index, f"server_node{_DELIM}{machine_id}{_DELIM}rack_id")

get detail from systemhealth or confstore
not from conf

Reply:
Done